### ISSUE TITLE
[v6] rc.1 validation issues

### ISSUE THREAD

**Issue by HriBB** [erikras/redux-form#1266]
Go to http://redux-form.com/6.0.0-rc.1/examples/syncValidation/ and press Submit

```
Unhandled promise rejection Object {username: "Required", email: "Required", age: "Required"}
  (anonymous function) @ es6.promise.js:119
                     A @ es6.promise.js:61
  (anonymous function) @ es6.promise.js:113
             e.exports @ _invoke.js:5
m.(anonymous function) @ _task.js:31
                     g @ _task.js:19
                     b @ _task.js:23
```

Can someone please confirm this. Thanks

**Comment by erikras** [comment]
Confirmed. 👍 Thanks.

**Comment by timothyallan** [comment]
Phew, I was getting all sorts of wonky validation issues last night with rc1 which never appeared in the last beta, this looks related.

**Comment by HriBB** [comment]
How can we fix this?

**Comment by erikras** [comment]
Good question. It _MUST_ be related to #1203.

**Comment by jcheroske** [comment]
I'm getting something that I think is related to this:
```
dynamics_browser.js:51 Uncaught (in promise) Object {email: "Invalid email address: foo@bar.com"}(anonymous function) 
@ dynamics_browser.js:51run 
@ zone.js?1467482723343:151zoneBoundFn 
@ tracer.js?1467482723343:163
```
This message just started appearing after the rc.1 upgrade. The submission error is still displayed correctly.

**Comment by HriBB** [comment]
https://github.com/erikras/redux-form/pull/1203/files#r69382571

@erikras any ideas on how to fix this?

**Comment by felisio** [comment]
@erikras When will be released this version to validation repaired? Thanks

**Comment by erikras** [comment]
Published fix in [`6.0.0-rc.2`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.2).

**Comment by kristian-puccio** [comment]
I'm still seeing this unfortunately. I am using async await with babel if that makes a difference.
